python/essex-config/essex_config/doc_gen/printer/markdown_printer.py

```diff
@@ -61,7 +62,7 @@ def _get_markdown(
                     prefix_annotation.prefix if prefix_annotation is not None else ""
                 )
 
-            if issubclass(field_type, BaseModel):
+            if is_pydantic_model(field_type):
```

Comment:
Do these changes have relationship with the 3.10 compat?

Reply:
Yuuup, issubclass with BaseModel as the second argument won't work with Python 3.10

Reply:
Yes, issubclass won't work with Pydantic models as the second argument.

Reply:
That's weird, I just tested it on my machine with and it works fine, is it a specific version of 3.10 that throws the error?
 
```python 
from pydantic import BaseModel
import sys
 
class SomeObj(BaseModel):
    a: int
    b: str
 
class SomeOtherObj:
    a: int
    b: str
 
 
print(sys.version)
print(issubclass(SomeObj, BaseModel))
print(issubclass(SomeOtherObj, BaseModel))
```
 
```
3.10.12 (main, Jul 29 2024, 16:56:48) [GCC 11.4.0]
True
False
```